layouts/shortcodes/gallery.html

```diff
@@ -0,0 +1,18 @@
+{{ $time := now.UnixNano }}
+{{ $id := delimit (slice "gallery" $time) "-" }}
+
+<div id="{{ $id }}">
+  {{ .Inner }}
+</div>
+
+<script>
+
+  document.addEventListener("DOMContentLoaded", function () {
+    var elem = document.querySelector('#{{ $id }}');
+    var pckry = new Packery(elem, {
+      percentPosition: true,
+      gutter: 5,
```

Comment:
Thought about that. Check the CSS though... the gutter needs to be baked in the CSS otherwise what you had defined would break the gallery. E.g. width 25% plus gutter doesn't allow for 4 columns...

Reply:
Ah... Yeah, i missed that. You're right. I've nothing else to add then. Looks good to me